src/server/v2.0/handler/model/artifact.go

```diff
@@ -84,6 +85,18 @@ func (a *Artifact) ToSwagger() *models.Artifact {
 			art.ScanOverview[key] = summary
 		}
 	}
+	if len(a.SBOMOverView) > 0 {
+		sbomOverview := &models.SBOMOverview{}
```

Comment:
move the init down to line 90 to avoid wasting memory.

Reply:
Updated